docs/templates/snakemake/README.md

```diff
@@ -0,0 +1,59 @@
+# {Snakemake pipeline template}
+
+## How to create your snakemake pipeline for APAeval
```

Comment:
Would be good to include a reference to the pilot benchmark, as it illustrates how this whole thing looks in practice

Reply:
done